packages/plugin-char/src/Char.ts

```diff
@@ -47,7 +48,12 @@ export class Char<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         const range = params.context.range;
         const text = params.text;
         const inline = this.editor.plugins.get(Inline);
-        const modifiers = inline.getCurrentModifiers(range);
+        let modifiers = inline.getCurrentModifiers(range);
+
+        const cleanModifier = range.start.previousSibling() === undefined || range.start.previousSibling().is(LineBreakNode);
```

Comment:
I generally avoid to write `undefined` unless in rare cases where it can be a somewhat valid value.

In this particular case I would have done:
```typescript
const previousSibling = range.start.previouspreviousSibling();
if (!previousSibling || previousSibling.is(LineBreakNode)) {
```

In the version with `undefined`, you have to "translate" in your brain the fact that `previousSibling` being `undefined` means that there is no previous sibling. The suggested code is more idiomatic IMO because it reads exactly what it means: "if there is no previous sibling or it is a LineBreakNode". You don't need to wrap your brain about anything in particular.

Reply:
Agree.